### PR TITLE
refactor(api): updated event log

### PR DESCRIPTION
**Pull request by madhavilosetty-intel**
## PR Checklist

- [ ] Unit Tests have been added for new changes
- [ ] API tests have been updated if applicable
- [ ] All commented code has been removed
- [ ] If you've added a dependency, you've ensured license is compatible with Apache 2.0 and clearly outlined the added dependency.

### CODE REVIEW

File: src/amt/amt/AMT.ts [hunk 1]
```diff
@@ -86,6 +86,18 @@ export class AMT {
     }
   }
 
+  MessageLog = (method: Methods.GET_RECORDS, messageId: string): string => {
```
Comment (rsdmike) [inline src/amt/amt/AMT.ts:89]: Do you have a test for this?

**Review by rsdmike — CHANGES_REQUESTED** [review]

File: src/amt/amt/AMT.ts (hunk 1 above)
Comment (madhavilosetty-intel) [reply src/amt/amt/AMT.ts]: added

File: src/routes/amt/eventLog.test.ts [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+  })
+  it('shout get system boot failure', () => {
+    const result = GetEventDetailStr(35, 0, [64, 10, 0, 0, 0, 0, 0, 0])
+    expect(result).toEqual('System boot failure')
+  })
+  it('should get system firmware started', () => {
+    const result = GetEventDetailStr(37, 0, [64, 10, 0, 0, 0, 0, 0, 0])
+    expect(result).toEqual('System firmware started (at least one CPU is properly executing).')
+  })
+  it('should get system firmware started', () => {
```
Comment (rsdmike) [inline src/routes/amt/eventLog.test.ts:76]: is this the right title?

File: src/routes/amt/eventLog.test.ts [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    expect(resSpy.json).toHaveBeenCalledWith({ error: 'Internal Server Error', errorDescription: 'Request failed during AMT EventLog.' })
+  })
+})
+
+describe('event details', () => {
+  it('should return authentication failed', () => {
+    const result = GetEventDetailStr(6, 2, [64, 3, 0, 0, 0, 0, 0, 0])
+    expect(result).toEqual('Authentication failed 3 times. The system may be under attack.')
+  })
+  it('should return system firware error', () => {
```
Comment (rsdmike) [inline src/routes/amt/eventLog.test.ts:48]: firware => firmware

**Review by rsdmike — CHANGES_REQUESTED** [review]

File: src/routes/amt/eventLog.test.ts (hunk 3 above)
Comment (madhavilosetty-intel) [reply src/routes/amt/eventLog.test.ts]: corrected

**Review by rsdmike — APPROVED** [review]

**Pull request merged**